server/index.js

```diff
@@ -47,8 +56,62 @@ app.post('/view', (req, res) => {
     yearWeek: calculateYearWeek(firstTimestamp),
     abandonFlag: Math.floor(calculateDuration(events) / (videoLength * (3 / 4)))
   };
-  res.send('data accepted');
   AbandonedTotal.addToTable(dbData);
+};
+
+// post route for incoming event packages (route name pending consensus with Event service)
+app.post('/view', (req, res) => {
+  // events object needs to change depending on whether or not its being tested with artillery
+  var events = JSON.parse(req.body.body).events;
+  // will replace fake route with actual location of video inventory service once ready for deployment
+  res.send('data accepted');
+  client.getAsync(events[0].videoId.toString())
```

Comment:
Also this is a large block of code, consider adding more descriptors to make it easier for others to follow.